### PR TITLE
Add Turkish translation

### PR DESCRIPTION
**Pull request by bitigchi**

### CODE REVIEW

**Comment by chrisbra** [comment]
Note there is also the installer: https://github.com/vim/vim/tree/master/nsis/lang and menus https://github.com/vim/vim/tree/master/runtime/lang that might need translation.

**Comment by bitigchi** [comment]
Good catch, thanks. Will look into them now.

**Comment by bitigchi** [comment]
I've also updated the menus, will look into the installer later. I tried to fix the tabs in the file during edit, but here they look even worse. Does it make a difference?

**Comment by brammool** [comment]
Looks like you are still working on this.  Let me know when it's ready to include.

File: src/po/tr.po [hunk 1]
```diff
@@ -0,0 +1,7284 @@
...
+msgstr "E516: Hiçbir arabellek silinmedi"
+
+msgid "E517: No buffers were wiped out"
+msgstr "E517: Hiçbir arabellek yok edilmedi"
+
+#, c-format
+msgid "%d buffer unloaded"
+msgid_plural "%d buffers unloaded"
+msgstr[0] "%d arabellek kaldırıldı"
+msgstr[1] "%d arabellek kaldırıldı"
```
Comment (k-takata) [inline src/po/tr.po:144]: Does Turkish have plural form?
You are translating singular (msgstr[0]) and plural (msgstr[1]) in the same form.

Comment (bitigchi) [reply src/po/tr.po:144]: No, both singular and plural forms use the same grammatical structure. Plural form affixes are not used when referencing the number of objects (1 arabellek, 2 arabellek etc.).

Comment (bitigchi) [reply src/po/tr.po]: > Looks like you are still working on this. Let me know when it's ready to include.

Got it! I will let you know!

**Comment by bitigchi** [comment]
> Looks like you are still working on this. Let me know when it's ready to include.

Got it! I will let you know.

**Comment by bitigchi** [comment]
It's okay to merge now. Thank you!

I will look into the installer later, since I believe I need to get my hands onto a Windows machine to check the strings. Please let me know if there is anything else.

**Comment by brammool** [comment]
Travis reports an error: tmp_gvim.desktop: error: file contains multiple keys named "Comment[tr]" in group "Desktop Entry"

**Comment by brammool** [comment]
Removing the entry from the .desktop.in files fixes that.

**Comment by brammool** [comment]
I'll include it, thanks.

**Pull request closed without merging**